docs/backbone.radio.md

```diff
@@ -0,0 +1,381 @@
+# Backbone Radio
+
+The Backbone Radio provides easy support for a number of messaging patterns for
+Backbone and Marionette. This is provided through two basic constructs:
+
+* Events - trigger events on a global object
+* Requests - a global request/reply implementation
+
+Radio takes these two constructs and adds the channel implementation - providing
+namespaces for events and requests. In short, Radio is a global, namespaced,
+message bus system designed to allow two otherwise unrelated objects to
+communicate and share information.
+
+Let's look at a simplified example in Marionette:
+
+```javascript
+var Marionette = require('backbone.marionette');
+
+var NotificationHandler = Marionette.Object.extend({
+  channelName: 'notify',
+
+  radioRequests: {
+    'show:success': 'showSuccessMessage',
+    'show:error': 'showErrorMessage'
+  },
+
+  radioEvents: {
+    'user:logged:in': 'showProfileButton',
+    'user:logged:out': 'hideProfileButton'
+  },
+
+  showSuccessMessage: function(message) {
+    // ...
+  },
+
+  showErrorMessage: function(message) {
+    // ...
+  },
+
+  showProfileButton: function(user) {
+    // ...
+  },
+
+  hideProfileButton: function(user) {
+    // ...
+  }
+});
+```
+
+In an unrelated module:
+
+```javascript
+var Radio = require('backbone.radio');
+var User = require('./models/user');
+
+var notifyChannel = Radio.channel('notify');
+var userModel = new User();
+
+// The following will call Notification.showErrorMessage(message)
+notifyChannel.request('show:error', 'A generic error occurred!');
+
+// The following will call Notification.showProfileButton(user)
+notifyChannel.trigger('user:logged:in', userModel);
```

Comment:
It's not about the channel name, it's about making the example more relevant and conceptually more correct from an API pov.